### PR TITLE
feat(breadcrumbs): added maxItems

### LINKED ISSUE #987: Enable truncation in the `Breadcrumb` component
### Description

A function that skips intermediate layers when a specified layer is reached.

- [Breadcrumb](https://ui.shadcn.com/docs/components/breadcrumb)

### Problem Statement/Justification

Depending on the application, there are pages with a deep hierarchy and there is a need to omit them.

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by hoshico**
Closes #987  

## Description

added `maxItems`, `itemsBeforeCollapse`, `itemsAfterCollapse` and `ellipsis`

## New behavior

If maxItems is specified and breadCrumbsItems exceeds that value, the ellipsis is applied

### CODE REVIEW

File: yamada-ui
Comment (nakayan5) [inline yamada-ui:1]: @hoshico 
did we ever user submodule ?

File: packages/components/breadcrumb/src/breadcrumb.tsx [hunk 1]
```diff
@@ -69,13 +97,64 @@ export const Breadcrumb = forwardRef<BreadcrumbProps, "nav">((props, ref) => {
...
+        gap,
+        isLastChild: count === index + 1,
+      }),
+    )
+
+    if (!items) return null
+
+    if (count < maxItems) {
+      return items
+    }
```
Comment (KenyaMasuko) [inline packages/components/breadcrumb/src/breadcrumb.tsx:113]:
```suggestion
    if (count <= maxItems) {
      return items
    }
```
If `count` equals `maxItems`, not all breadcrumb items are shown. So, should the conditional branch include `=`, shouldn't it?

**Review by KenyaMasuko — COMMENTED** [review]
Are there no tests for this feature change?
And please check about conditional sentence!

**Comment by hoshico** [comment]
I'll close this for now as you've suggested starting with just the ellipsis

**Pull request closed without merging**